src/plugins/dbus/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ add_plugin (dbus
 	INCLUDE_DIRECTORIES
 		${DBUS_INCLUDE_DIR}
 		${DBUS_ARCH_INCLUDE_DIR}
+	LINK_ELEKTRA
+		elektra-io-adapter-dbus
```

Comment:
Does sending dbus messages also work without any support from applications?

Would be good to have some documentation/tutorials/...

Reply:
Yes it works without modifications. If an I/O binding is set by an application using `elektraIoSetBinding()` the plugin uses it to send dbus messages.

I will add this to the dbus plugin readme.

Reply:
Well, after reading the API docs for `dbus_connection_send()` I discovered that it already operates asynchonously. So the adapter is only needed in the dbusrecv plugin.